packages/mui-material/src/Select/SelectInput.js

```diff
@@ -487,12 +487,15 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
     ...MenuProps.slotProps?.paper,
   };
 
+  const listboxId = useId();
+
   return (
     <React.Fragment>
       <SelectSelect
         ref={handleDisplayRef}
         tabIndex={tabIndex}
-        role="button"
+        role="combobox"
```

Comment:
Shouldn't this be considered a breaking change, so it should be a minor version change instead of a patch change?
v5.14.11 -> v5.15.0

Reply:
Hi! It's an a11y bug fix; the `"button"` role was incorrect, so I think it falls under a patch version. Also, this should keep functionality intact; tests should only be impacted by how the component is selected. Please let me know if this is not the case for you 😊

Reply:
This was an absolutely awful decision. The notes on 5.14.12 did not detail that this would be a breaking change. It should have been called out immediately.

Reply:
Hey @Lori-Becker and @mrharispe, I'm sorry for the inconvenience this might've caused. We will learn from this and be more careful moving forward. Thank you for using the library and providing feedback.